migrations/command-line/apology_gems.js

```diff
@@ -1 +1 @@
-db.users.update({ _id: { $in: [''] } }, { $inc: { balance: 0.5 } }, { multi: true });
+db.users.updateOne({ _id: { $in: [''] } }, { $inc: { balance: 0.5 } }, { multi: true });
```

Comment:
not that this migration really needs to be updated at all (we can probably get rid of it tbh), but an `$in` query should be a many, not a one

Reply:
done!